common/models/user.js

```diff
@@ -38,6 +42,21 @@ module.exports = function(User) {
   // see:
   // https://github.com/strongloop/loopback/issues/1137#issuecomment-109200135
   delete User.validations.email;
+  // Updated workaround for email validation
+  // see:
+  // https://github.com/strongloop/loopback/issues/1137#issuecomment-271437926
+  User.validate('email', function( err ) {
+    if (this.email && !isEmail(this.email)) {
+      debug(err);
+    }
+  }, {
+    message: 'Email format is invalid, please check again'
```

Comment:
Let's change this to 'Please check to make sure the email is a valid email address.'